README.md

```diff
@@ -121,7 +123,7 @@ In order to make testing out features more convenient, you can toggle feature fl
 
 To enable a feature flag temporarily during development, append it as a query string in the URL:
 ```
-http://localhost:5000/somepage?my_feature=true
+http://localhost:8080/somepage?my_feature=true
```

Comment:
Both :8080 and :5000 appear in the readme; we should be consistent. It's :8080 on my machine so let's go with that